code/datums/manufacturing.dm

```diff
@@ -3081,6 +3081,16 @@ ABSTRACT_TYPE(/datum/manufacture/pod/weapon)
 	time = 20 SECONDS
 	category = "Tool"
 
+/datum/manufacture/pod/weapon/efif1
+	name = "EFIF-1 Construction System"
+	item_requirements = list("metal_superdense" = 50,
+							 "claretine" = 20,
+							 "electrum" = 10)
```

Comment:
These should not be specific material requirements

Reply:
They are specific material requirements for a particular purpose - using conductive_high cannot distinguish between claretine and electrum, and electrum is a deliberate step to increase complexity of production.